systemtest/src/test/resources/upgrade/StrimziDowngradeST.yaml

```diff
@@ -30,6 +31,7 @@
   toExamples: strimzi-0.33.0
   urlFrom: HEAD
   urlTo: https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.33.0/strimzi-0.33.0.zip
+  kafkaVersionsUrlFrom: HEAD
```

Comment:
Should this be a local file path instead?

Reply:
As we have in `urlFrom`, if it is `HEAD`, we are getting it from the local classes or files -> as you can see in `getKafkaWithVersionFromUrl` -> if it is `HEAD`, the specific version from already existing `TestKafkaVersion` is returned insted of grabbing it from the URL.